compiler/src/dotty/tools/dotc/ast/untpd.scala

```diff
@@ -526,7 +526,7 @@ object untpd extends Trees.Instance[Untyped] with UntypedTreeInfo {
       case Function(args, body) =>
         this(this(x, args), body)
       case InfixOp(left, op, right) =>
-        this(this(x, left), right)
+        this(this(this(x, left), op), right)
       case PostfixOp(od, op) =>
```

Comment:
`UntypedTreeAccumulator` (and its cousing `UntypedDeepFolder`) is used only in the repl code (`CompilingInterpreter.scala`).
Changed `{Post, Pre}fixOp` as well.
PTAL.